### DISCUSSION TITLE
Mitre Enginuity Mapping

### DISCUSSION THREAD

**Discussion by robmoffat** [finos/common-cloud-controls#206] [Ideas]
This is what I mentioned in the Duplication Reduction meeting yesterday:

https://mitre-engenuity.org/cybersecurity/center-for-threat-informed-defense/our-work/nist-800-53-control-mappings/

"a comprehensive and open, curated set of mappings between 800-53 controls and ATT&CK techniques."

Something else to check out, a bit like @mlysaght2017 mentions in https://github.com/finos/common-cloud-controls/discussions/203

**Comment by damienjburks** [comment]
@robmoffat - are we good to close this out?

**Comment by robmoffat** [comment]
sure, we have a PR of this now
